app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,88 @@
+.overlay-container {
+  background: #000000;
+  height: 23rem;
+  overflow: hidden;
+  padding-left: 15%;
+  position: relative;
+}
+
+.overlay-container .overlay {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 4rem 2rem 2rem 1rem;
+  color: white;
+  margin: 0;
+  width: 30%;
+}
+
+p {
+  font-size: 100%;
+}
+
+.addstory {
+  margin-top: 1.25rem;
+}
+
+.f-topbar-fixed {
+  border-bottom: 10px;
+  padding-top: 0;
+}
+
+.top-bar {
+  box-shadow: 0 4px #cf6e0e;
```

Comment:
Color literals like `#cf6e0e` should only be used in variable declarations; they should be referred to via variable everywhere else.